packages/turf-union/index.js

```diff
@@ -1,10 +1,13 @@
-import { GeoJSONReader, GeoJSONWriter, UnionOp } from 'turf-jsts';
+import martinez from 'martinez-polygon-clipping';
+import { getGeom } from '@turf/invariant';
+import { multiPolygon, polygon } from '@turf/helpers';
 
 /**
  * Takes two or more {@link Polygon|polygons} and returns a combined polygon. If the input polygons are not contiguous, this function returns a {@link MultiPolygon} feature.
```

Comment:
Note: I've added more links to the Documentation YAML config.

- `{@link Polygon|polygons}` => `{@link (Multi)Polygon(s)}`

https://github.com/Turfjs/turf/blob/master/documentation.yml#L177